recipes/yandex-query-client/0001-remove-typing.patch

```diff
@@ -0,0 +1,11 @@
+diff -ruN yandex_query_client-0.1.3/pyproject.toml yandex_query_client-0.1.3-patch/pyproject.toml
+--- yandex_query_client-0.1.3/pyproject.toml	2020-02-02 01:00:00.000000000 +0100
++++ yandex_query_client-0.1.3-patch/pyproject.toml	2024-04-18 16:34:30.554880474 +0200
+@@ -21,7 +21,6 @@
+   "pandas>=1.2.5",
+   "python-dateutil>=2.8.2",
+   "requests>=2.31.0",
+-  "typing>=3.7.4.3",
```

Comment:
Patching to remove `typing` from the `pyproject.toml` so that `pip check` can pass.  `typing` is an empty metapackage on conda-forge:
https://github.com/conda-forge/typing-feedstock/blob/main/recipe/meta.yaml

Reply:
I requested to have typing removed:
https://github.com/yandex-cloud/yandex-query-client/issues/1